_docs/individual-payments.md

```diff
@@ -440,6 +462,68 @@ orange
 
 {% include corridors/xof-bank.md recipient_type='individual' %}
 
+# Cameroon
+
+## XAF::Mobile
+
+For mobile payouts to Cameroon, please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "mobile_provider": "orange", // lowercase, see provider values below
+  "phone_number": "674044436", // mobile number in local country format
+  "country": "CM" // "CM" for Cameroon
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xaf-mobile-details" raw=data-raw %}
+
+The valid `mobile_provider` values for Cameroon are:
+
+{% capture data-raw %}
+```
+orange
+mtn
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xaf-mobile-providers" raw=data-raw %}
+
+# Guinea Conakry
+
+## GNF::Mobile
+
+For mobile payouts to Guinea Conakry, please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "mobile_provider": "mtn", // lowercase, see provider values below
+  "phone_number": "444044436", // mobile number in local country format
+  "country": "GN" // "GN" for Guinea
```

Comment:
We only allow `GN` as country for `GNF::Mobile` payouts so we don't need to ask the client to specify it.

Reply:
For Guinea there's no possibility for the client to set another country so I'm wondering if this can't be removed altogether...

Reply:
@cesarecamurani I did put it as an optional value, but sure we can remove it